junebug/channel.py

```diff
@@ -178,11 +184,15 @@ def _get_status(self):
                 key=status_values.get)
         except ValueError:
             # No statuses
-            returnValue({})
+            status = None
```

Comment:
I'm wondering whether `None` is the most sensible return value here, but I think it is. Just flagging this for thinking about.